plugins/experimental/cookie_remap/strip.cc

```diff
@@ -228,7 +228,7 @@ get_stripped(const char *in, ssize_t in_len, char *out, int *out_len, unsigned i
   retval = stripped_core(r, in_end, &w, out_end, flags);
 
   /* null terminate */
-  out_end += out_end ? 1 : 0;            /* undo decrement at start */
+  out_end++;                             /* undo decrement at start */
```

Comment:
Should we add a parameter check at line 192 that if `out` is non-nullptr, then `*out_len > 0` ?

Reply:
I think, in that case, there will be no space for the `out_end++`, right?